### PR TITLE
Configure New Relic extension in code.

### PR DESCRIPTION
**Pull request by DFurnes**
### What does this PR do?

This pull request configures the New Relic extension in code so that we can more reliably control whether it's on or off in different environments. 

### Any background context you want to provide?

This will allow us to disable New Relic on our QA instances and still have the extension built when promoting that Heroku slug into production. 📛

Also, a reminder to [update your Homestead environment](https://git.io/fpdne) if you haven't already (since Composer will now start checking whether `ext-newrelic` is around when running installs/updates). If you don't have time for a rebuild right now, you can run the agent install steps from `after.sh` by hand in your terminal!

### What are the relevant tickets/cards?

Refs DoSomething/infrastructure#60.

### Checklist

* [ ] Documentation added for new features/changed endpoints.
* [ ] Added appropriate feature/unit tests.
* [ ] Added screenshot to PR description of related front-end updates on **small** screens.
* [ ] Added screenshot to PR description of related front-end updates on **medium** screens.
* [ ] Added screenshot to PR description of related front-end updates on **large** screens.

### CODE REVIEW

**Comment by DFurnes** [comment]
I've updated dependencies in 8039b74 to commit this change into the lockfile. The diff:

```
Package operations: 2 installs, 34 updates, 0 removals
  - Updating doctrine/dbal (v2.8.0 => v2.9.0)
  - Updating symfony/polyfill-mbstring (v1.9.0 => v1.10.0)
  - Updating symfony/var-dumper (v3.4.16 => v3.4.20)
  - Updating psr/log (1.0.2 => 1.1.0)
  - Updating symfony/debug (v3.4.16 => v3.4.20)
  - Installing symfony/contracts (v1.0.2)
  - Updating symfony/console (v3.4.16 => v3.4.20)
  - Updating nikic/php-parser (v4.0.4 => v4.1.0)
  - Updating jakub-onderka/php-console-highlighter (v0.3.2 => v0.4)
  - Updating psy/psysh (v0.9.8 => v0.9.9)
  - Updating symfony/routing (v3.4.16 => v3.4.20)
  - Updating symfony/process (v3.4.16 => v3.4.20)
  - Updating symfony/polyfill-ctype (v1.9.0 => v1.10.0)
  - Updating paragonie/random_compat (v2.0.17 => v9.99.99)
  - Updating symfony/polyfill-php70 (v1.9.0 => v1.10.0)
  - Updating symfony/http-foundation (v3.4.16 => v3.4.20)
  - Updating symfony/event-dispatcher (v4.1.5 => v4.2.1)
  - Updating symfony/http-kernel (v3.4.16 => v3.4.20)
  - Updating symfony/finder (v3.4.16 => v3.4.20)
  - Updating egulias/email-validator (2.1.6 => 2.1.7)
  - Updating symfony/translation (v4.1.5 => v4.2.1)
  - Updating nesbot/carbon (1.34.0 => 1.36.1)
  - Updating monolog/monolog (1.23.0 => 1.24.0)
  - Updating league/flysystem (1.0.47 => 1.0.49)
  - Updating laravel/framework (v5.5.43 => v5.5.44)
  - Updating laravel/tinker (v1.0.7 => v1.0.8)
  - Updating filp/whoops (2.2.1 => 2.3.1)
  - Updating mockery/mockery (0.9.9 => 0.9.10)
  - Updating symfony/filesystem (v4.1.5 => v4.2.1)
  - Installing ralouphie/getallheaders (2.0.5)
  - Updating guzzlehttp/psr7 (1.4.2 => 1.5.2)
  - Updating lcobucci/jwt (3.2.4 => 3.2.5)
  - Updating league/oauth2-client (2.3.0 => 2.4.1)
  - Updating giggsey/locale (1.5 => 1.6)
  - Updating phpseclib/phpseclib (2.0.11 => 2.0.12)
  - Updating composer/ca-bundle (1.1.2 => 1.1.3)
```

**Comment by DFurnes** [comment]
Oh, of course – Wercker needs this for our CI builds now too. Let me fix that up... stay tuned.

**Comment by DFurnes** [comment]
Alright, the build is green now! ✅ Ready for review!

File: public/.user.ini [hunk 1]
```diff
@@ -1,3 +1,7 @@
 memory_limit = 256M
 post_max_size = 10M
 upload_max_filesize = 10M
+
+# Custom New Relic settings, in addition to Heroku's
+# defaults, which are set here <https://git.io/fpazT>:
+newrelic.enabled = "${NEW_RELIC_ENABLED}"
```
Comment (weerd) [inline public/.user.ini]: Is this an env var we should add to .env.example?

Comment (DFurnes) [reply public/.user.ini]: It should default to false, I believe, and we don't want to connect to New Relic on locals. I can add it for visibility, though, if you think that's a good idea!

Comment (weerd) [reply public/.user.ini]: oh wait, is this isn't pulled from the Heroku env variable settings in each app right? Then I guess it's not needed.

Comment (DFurnes) [reply public/.user.ini]: Yeah, this is pulled from Heroku environment variables that are automatically set in [our `laravel_app` Terraform module](https://github.com/DoSomething/infrastructure/blob/cbd3bde56f64ec821187bd3b299a252031494424/shared/laravel_app/main.tf#L94-L98). I think my preference is to leave out of `.env.example` (unless we end up seeing issues) just so folks don't accidentally set it to `true` and get errors.

**Pull request merged**